README.md

```diff
@@ -47,7 +45,7 @@ yarn add dgraph-js-http
 or npm:
 
 ```sh
-npm install dgraph-js-http --save
```

Comment:
hasn't been needed in modern versions of npm for a long time, see https://stackoverflow.com/questions/19578796/what-is-the-save-option-for-npm-install